inference-engine/src/preprocessing/ie_preprocess_gapi_kernels_simd_impl.hpp

```diff
@@ -783,6 +783,134 @@ inline void copyRow_32F_impl(const float in[], float out[], int length) {
     }
 }
 
+// Resize (bi-linear, 32FC1)
+static inline void calcRowLinear_32FC1(float *dst[],
+                                       const float *src0[],
+                                       const float *src1[],
+                                       const float  alpha[],
+                                       const int    mapsx[],
+                                       const float  beta[],
+                                       const Size& inSz,
+                                       const Size& outSz,
+                                               int lpi) {
+    bool xRatioEq1 = inSz.width == outSz.width;
+    bool yRatioEq1 = inSz.height == outSz.height;
+
+#if CPU_SIMD
+    const int nlanes = v_float32::nlanes;
+#endif
+
+    if (!xRatioEq1 && !yRatioEq1) {
+        for (int l = 0; l < lpi; l++) {
+            float beta0 = beta[l];
+            float beta1 = 1 - beta0;
+
+            int x = 0;
+
+#if CPU_SIMD
+            for (; x <= outSz.width - nlanes; x += nlanes) {
+                v_float32 alpha0 = vx_load(&alpha[x]);
+                //  v_float32 alpha1 = 1.f - alpha0;
+
+                v_float32 s0l, s0h, s00, s01;
+                v_gather_pairs(src0[l], mapsx, x, s0l, s0h);
+                v_deinterleave(s0l, s0h, s00, s01);
+
+                //  v_float32 res0 = s00*alpha0 + s01*alpha1;
+                v_float32 res0 = v_fma(s00 - s01, alpha0, s01);
+                
+                v_float32 s1l, s1h, s10, s11;
```

Comment:
The same comment about naming, lo, hi or low and high look better

Reply:
done